pontoon/insights/static/js/insights.js

```diff
@@ -56,90 +106,104 @@ var Pontoon = (function (my) {
             borderWidth: item.name === 'All' ? 3 : 1,
             pointBackgroundColor: color,
             pointHitRadius: 10,
-            pointRadius: 4,
+            pointRadius: 3.25,
```

Comment:
Nit: Could you please change all values of pointRadius to `3.25` across all charts?